docs/user/user/pings/index.md

```diff
@@ -52,36 +92,126 @@ See [Using the Experiments API](../experiments-api.md) on how to record experime
 ```
 
 ### The `client_info` section
+
 The following fields are included in the `client_info` section.
 Optional fields are marked accordingly.
 
-| Field name | Type | Description |
-|---|---|---|
-| `app_build` | String | The build identifier generated by the CI system (e.g. "1234/A"). For language bindings that provide automatic detection for this value, (e.g. Android/Kotlin), in the unlikely event that the build identifier can not be retrieved from the OS, it is set to `inaccessible`. For other language bindings, if the value was not provided through configuration, this metric gets set to `Unknown`. |
-| `app_channel` | String | *Optional* The product-provided release channel (e.g. "beta") |
-| `app_display_version` | String | The user-visible version string (e.g. "1.0.3"). The meaning of the string (e.g. whether semver or a git hash) is application-specific. In the unlikely event this value can not be obtained from the OS, it is set to "inaccessible". If it is accessible, but not set by the application, it is set to "Unknown". |
-| `architecture` | String | The architecture of the device (e.g. "arm", "x86") |
-| `client_id` | UUID |  *Optional* A UUID identifying a profile and allowing user-oriented correlation of data |
-| `device_manufacturer` | String | *Optional* The manufacturer of the device |
-| `device_model` | String | *Optional* The model name of the device. On Android, this is [`Build.MODEL`], the user-visible name of the device. |
-| `first_run_date` | Datetime | The date of the first run of the application, in local time and with day precision, including timezone information. |
-| `os` | String | The name of the operating system (e.g. "linux", "Android", "ios") |
-| `os_version` | String | The user-visible version of the operating system (e.g. "1.2.3") |
-| `android_sdk_version` | String | *Optional*. The Android specific SDK version of the software running on this hardware device (e.g. "23") |
-| `telemetry_sdk_build` | String | The version of the Glean SDK |
-| `locale` | String | *Optional*. The locale of the application during initialization (e.g. "es-ES"). If the locale can't be determined on the system, the value is "und", to indicate "undetermined". |
-
-All the metrics surviving application restarts (e.g. `client_id`, ...) are removed once the application using the Glean SDK is uninstalled.
+#### `app_build`
+
+_Type: [String](../../reference/metrics/counter.md),
+Lifetime: [Ping](../../reference/yaml/metrics.md#ping-default)_
+
+The build identifier generated by the CI system (e.g. "1234/A"). For language bindings that provide
+automatic detection for this value, (e.g. Android/Kotlin), in the unlikely event that
+the build identifier can not be retrieved from the OS, it is set to `inaccessible`.
+For other language bindings, if the value was not provided through configuration,
+this metric gets set to `Unknown`.
```

Comment:
huh, I guess we should have standardized this better...

Reply:
(for some it's lowercase `unknown`. I might file a bug so we make it the same value everywhere, don't bother about this in the docs though)

Reply:
Nice catch :)